### DISCUSSION TITLE
How can I seamlessly integrate an API into my DeStack powered website for communication with back-end and enhanced functionality?

### DISCUSSION THREAD

**Discussion by vatsalxbyte** [LiveDuo/destack#108] [General]
How can Integrating an API into your DeStack-powered website facilitates seamless communication with the back-end and real-time data exchange

**Comment by LiveDuo** [comment]
@vatsalxbyte This is currently not supported by Destack.

A simple workaround would be to:
1. Use a custom placeholder syntax as text in Destack components eg. {data} or %data%
2. Dynamically replace that placeholders after the component is loaded

At some point this will be supported out of the box but it's not there yet.
